controllers/homeController.js

```diff
@@ -76,6 +87,31 @@ var findCommonChampions = function(championsA, championsB) {
     return commonChampions
 }
 
+var compareCommonChampions = function(commonChampions) {
+    var aCount = 0;
+    var bCount = 0;
+    commonChampions.forEach(function(championTuple) {
```

Comment:
Thanks for point this out! I expect I'll make a lot of these kinds of mistakes, so don't hesitate to point all of them out.